kube/services/revproxy/gen3.nginx.conf/arborist-service.conf

```diff
@@ -31,7 +31,7 @@ location = /gen3-authz {
 # authorization endpoint
 # https://hostname/authz?resource=programs/blah&method=acb&service=xyz
 #
-location ~ /authz/? {
+location = /authz/? {
```

Comment:
It's been a minute since I've written nginx rules. Does the `~` imply regex? Does the `?` lose meaning if `=` is just a string comparison?

Reply:
i think we're replacing "contains /authz/<...>" with "starts with /authz/<...>"

Reply:
you probably knew that haha
curious about the answer as well